api/envoy/api/v2/core/base.proto

```diff
@@ -182,6 +185,12 @@ message Node {
   // See :ref:`the list of features <client_features>` that xDS client may
   // support.
   repeated string client_features = 10;
+
+  // Known listening ports on the node as a generic hint to the management server
+  // for filtering :ref:`listeners <config_listeners>` to be returned. For example,
+  // if there is a listener bound to port 80, the list can optionally contain the
+  // SocketAddress `(0.0.0.0,80)`. The field is optional and just a hint.
+  repeated Address listening_addresses = 11;
```

Comment:
This is not implemented yet in this PR? so hide from doc? or do you plan to implement them?

Reply:
Not sure about what is "not implemented". We are planning to use this new field in our xDS client (proxyless gRPC) as a hint to the xDS server for particular listener(s). I don't see any reason hide it in the docs. 

Also note that we had some offline discussion with @htuch about this and this was the agreed upon approach.

Reply:
@lizan the user specifics it in bootstrap, and this then gets automagically dumped into each xDS stream (we merge `Node` messages).

Reply:
@htuch ah ok I see how it works now. Thanks.